deps/rabbit/test/backing_queue_SUITE.erl

```diff
@@ -1710,7 +1731,7 @@ with_fresh_variable_queue(Fun, Mode) ->
 set_queue_mode(Mode, VQ) ->
     VQ1 = rabbit_variable_queue:set_queue_mode(Mode, VQ),
     S1 = variable_queue_status(VQ1),
-    assert_props(S1, [{mode, Mode}]),
+%    assert_props(S1, [{mode, Mode}]),
```

Comment:
Remove the assert since it's ignored by the queue. Still set queue mode because we want this to be tested.